AcademicFlow.Managers.Contracts/IManagers/ICourseManager.cs

```diff
@@ -0,0 +1,37 @@
+﻿using AcademicFlow.Domain.Contracts.Entities;
+using AcademicFlow.Domain.Contracts.Enums;
+using AcademicFlow.Domain.Entities;
+using AcademicFlow.Managers.Contracts.Models.CourseModels;
+
+namespace AcademicFlow.Managers.Contracts.IManagers
+{
+    public interface ICourseManager
+    {
+        int? AddCourse(Course course);
+        
+        void DeleteCourse(int id);
+        
+        Course? GetCourseById(int id);
+
+        void UpdateCourse(Course course);
```

Comment:
why the method is not async? look at other services, it should be async and return Task